security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -735,3 +744,16 @@ func (sc *SecretManagerClient) UpdateConfigTrustBundle(trustBundle []byte) error
 func (sc *SecretManagerClient) mergeConfigTrustBundle(rootCert []byte) []byte {
 	return pkiutil.AppendCertByte(sc.getConfigTrustBundle(), rootCert)
 }
+
+func (sc *SecretManagerClient) GetCARootPath() string {
```

Comment:
Doc string for public method.

Reply:
same for others.

Reply:
Doc added to public function and variable. GetCARootPath and SetCARootPath no longer needed with change to get the OS CA file path on init